parsing/pcoq.ml

```diff
@@ -59,7 +59,7 @@ module type S =
 
   type coq_parsable
 
-  val coq_parsable : ?file:Loc.source -> char Stream.t -> coq_parsable
+  val coq_parsable : ?loc:Loc.t -> char Stream.t -> coq_parsable
```

Comment:
Just an idea, you could keep the `?file` arg and add another optional `?loc` arg as to be more compatbile; likely not worth it.

Reply:
Given that I've added an API `Loc.initial` with just one argument, the source, I think that creating a loc out of a source is simple enough that this API can just take the loc.

Reply:
Sounds good then.